roles/firewall/tasks/main.yml

```diff
@@ -71,6 +71,13 @@
   tags:
     - firewall
     - monitoring
+    
+- name: Allow ZMQ connections
+  ufw:  rule=allow
+        port=5550
+        proto=tcp
```

Comment:
Better discuss this a little bit more, yes. I'm not really keen on opening up that port for the entire world to come knocking, we need to find some sort of solution to have it more secure (because eventually someone will start knocking on that port).

For the non-static IP based lab, wouldn't it be possible to use some sort of DynDNS system (not DynDNS per-se...)?